custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -1374,7 +1374,7 @@ BEGIN
 		'baby_scale_usable AS infra_baby_weighing_scale, ' ||
 		'flat_scale_usable AS infra_flat_weighing_scale, ' ||
 		'GREATEST(adult_scale_available, adult_scale_usable) AS infra_adult_weighing_scale, ' ||
-		'GREATEST(baby_scale_usable, flat_scale_usable) AS infra_infant_weighing_scale, ' ||
+		'GREATEST(baby_scale_available, flat_scale_available, baby_scale_usable) AS infra_infant_weighing_scale, ' ||
```

Comment:
usable is from app v2, available is from app v1. flat_scale_usable doesn't exist in the app so removing it here. Keeping it in the data source because we don't support removing columns easily